code/game/objects/items/weapons/tanks/jetpack.dm

```diff
@@ -61,13 +61,22 @@
 /obj/item/tank/jetpack/verb/toggle_rockets()
 	set name = "Toggle Jetpack Stabilization"
 	set category = "Object"
-	src.stabilization_on = !( src.stabilization_on )
-	to_chat(usr, "You toggle the stabilization [stabilization_on? "on":"off"].")
+
+	proc_toggle_rockets(usr)
+
+/obj/item/tank/jetpack/proc/proc_toggle_rockets(mob/user, var/list/message_mobs)
```

Comment:
So, all this does is toggle stabilization_on.

Why isn't proc/proc_toggle used instead of this proc?